wasp/wasp.py

```diff
@@ -116,18 +123,20 @@ def __init__(self):
 
         # TODO: Eventually these should move to main.py
         for app, qr in ( (ClockApp, True),
-                         (StepCounterApp, True),
-                         (StopwatchApp, True),
-                         (HeartApp, True),
-                         (FlashlightApp, False),
-                         (SettingsApp, False),
-                         (TestApp, False) ):
-            try:
-                self.register(app(), qr)
-            except:
-                # Let's not bring the whole device down just because there's
-                # an exception starting one of the apps...
-                pass
+             (StepCounterApp, True),
+			 (StopwatchApp, True),
+			 (HeartApp, True),
+			 (FlashlightApp, False),
+			 (SettingsApp, False),
+             (TestApp, False),
+             (MusicPlayerApp, False)):
```

Comment:
As mentioned, I'd rather not enable the MusicPlayer until I've had a change to examine RAM usage... but I'm happy to merge it before I do that.

Don't add it here... add it to docs/apps.rst and wasp/boards/simulator/main.py instead (simulator has no RAM limit so it becomes a showcase for all the upstream applications).

I am looking at better ways to enable/disable applications but for now we are putting optional apps into the app library.